docs/ops/detection/ExperimentalDetectronPriorGridGenerator_6.md

```diff
@@ -0,0 +1,97 @@
+## ExperimentalDetectronPriorGridGenerator <a name="ExperimentalDetectronPriorGridGenerator"></a> {#openvino_docs_ops_detection_ExperimentalDetectronPriorGridGenerator_6}
+
+**Versioned name**: *ExperimentalDetectronPriorGridGenerator-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronPriorGridGenerator* operation generates prior grids of specified sizes.
+
+**Attributes**:
+
+* *flatten*
+
+    * **Description**: *flatten* attribute specifies whether the output tensor should be 2D or 4D.
+    * **Range of values**:
+      * `true` - the output tensor should be 2D tensor
+      * `false` - the output tensor should be 4D tensor
+    * **Type**: boolean
+    * **Default value**: true
+    * **Required**: *yes*
```

Comment:
It is optional attribute, see https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/vpu/graph_transformer/src/stages/exp_priorgridgenerator.cpp#L77

Reply:
Yes, updated.